src/Bandits/BanditVariationIndexer.php

```diff
@@ -26,11 +26,16 @@ public function __unserialize(array $data): void
      */
     private array $banditFlags = [];
 
+
+    private function __construct()
```

Comment:
switching to static helper methods for construction as I wanted to be able to create an empty indexer without having to worry about it throwing an exception